modules/cms/widgets/AssetList.php

```diff
@@ -662,7 +663,11 @@ protected function checkUploadPostback()
             /*
              * Accept the uploaded file
              */
-            $uploadedFile->move($this->getCurrentPath(), $uploadedFile->getClientOriginalName());
+            $uploadedFile = $uploadedFile->move($this->getCurrentPath(), $uploadedFile->getClientOriginalName());
+
+            if (Config::get('cms.defaultMask.file')) {
+                @chmod($uploadedFile->getRealPath(), octdec(Config::get('cms.defaultMask.file')));
```

Comment:
@kmarty Just a minor change - we shouldn't suppress the error here.
```suggestion
                chmod($uploadedFile->getRealPath(), octdec(Config::get('cms.defaultMask.file')));
```